src/sign.ts

```diff
@@ -89,7 +91,7 @@ export class Signer {
       ],
     };
 
-    const certificateB64 = enc.base64Encode(certificate);
+    const certificateB64 = enc.base64Encode(certificate[0]);
```

Comment:
Nit should this also be 
```suggestion
    const leafCertificate = certificate[0];
    const leafCertificateB64 = enc.base64Encode(leafCertificate);
```